cli/src/app.rs

```diff
@@ -347,7 +347,49 @@ pub fn add_subcommands(command: Command) -> Command {
                             .help("Name for the new group")
                             .required(true),
                     ),
-                ),
+                )
+                .subcommand(
+                    Command::new("members").about("List group members").args(&[
```

Comment:
1. This command can do more than simply `List group members` and the about message should reflect that
2. For consistency with our other commands, `members` should not be plural (this will require changing more than just this line)

```suggestion
                    Command::new("member").about("Manage group members").args(&[
```

Reply:
> For consistency with our other commands, members should not be plural (this will require changing more than just this line)

Yeah I found myself mixing up members/member while testing myself. Sole reason for choosing members was the API naming.

Reply:
Right. Plural is standard in the API (for some reason that I don't understand). But we should keep singular as the standard in CLI.

I guess we could add aliases to allow both (we do that for `phylum project` for historical reasons). But I'd rather not add that complexity everywhere.